.gitignore

```diff
@@ -36,8 +36,13 @@ bazel-*
 # Compiled files #
```

Comment:
We've integrated the Rust IDE sync tool into `typedb-protocol`, which can result in the generation of `target/`, `Cargo.toml` and `Cargo.lock`.